apps/aeutil/lib/serialization.ex

```diff
@@ -247,4 +248,32 @@ defmodule Aeutil.Serialization do
   end
 
   def deserialize_value(value, _), do: value
+
+  def serialize_txs_info_to_json(txs_info) when is_list(txs_info) do
+    serialize_txs_info_to_json(txs_info, [])
+  end
+
+  def serialize_txs_info_to_json([h | t], acc) do
+    json_response_struct = %{
+      tx: %{
+        sender: Account.base58c_encode(h.sender),
+        recipient: Account.base58c_encode(h.payload.receiver),
+        amount: h.payload.amount,
+        fee: h.fee,
+        nonce: h.nonce,
+        vsn: h.payload.version
+      },
+      block_height: h.block_height,
+      block_hash: Header.base58c_encode(h.block_hash),
+      hash: SignedTx.base58c_encode_root(h.txs_hash),
+      signatures: [base64_binary(h.signature, :serialize)]
+    }
+
+    acc = acc ++ [json_response_struct]
+    serialize_txs_info_to_json(t, acc)
+  end
+
+  def serialize_txs_info_to_json([], acc) do
```

Comment:
add `@spec`

Reply:
Its a recursive function, so the spec of first function applies to the rest as well.